test/units/modules/network/cnos/fixtures/l3_interface_config.cfg

```diff
@@ -0,0 +1,339 @@
+!
+version "10.8.0.42"
```

Comment:
This doesn't have to be a complete configuration... just the relevant sections should be enough

Reply:
I will make it short. Actually it was easy for me to copy paste the running config. I will change it.